packages/visualizations/src/components/Map/types.ts

```diff
@@ -128,6 +111,13 @@ export interface ChoroplethFixedTooltipDescription {
 
 export type MapRenderTooltipFunction = DebouncedFunc<(f: Feature) => string>;
 
+export type ComputeTooltipFunction = (
+    f: Feature,
```

Comment:
I'm nitpicking, sorry, but could we name the parameter `feature` to be more explicit?

Reply:
Ok 👍 I've just changed it !